### ISSUE TITLE
Dynamic currency

### ISSUE THREAD

**Issue by GENL** [mtvbrianking/laravel-mtn-momo#15]
I want to be able to pass the currency in parameter of the method "transact". Can you please add this functionality? I think is more flexible than just hard writing it in the .env file.

**Comment by mtvbrianking** [comment]
@GENL I intended on keeping transact method signature small, since currency in operation will always be constant

**Comment by GENL** [comment]
This can be an optional parameter whose default value is the value of the .env file.
I also want to know if we can use this plugin for different use cases of MOMO, and not only "request to pay".

**Comment by mtvbrianking** [comment]
I get, I'll add it tomorrow.

**Comment by GENL** [comment]
Ok. Thanks!

**Comment by mtvbrianking** [comment]
Implemented setters and getters in release [v0.2.0](https://github.com/mtvbrianking/laravel-mtn-momo/releases/tag/v0.2.0)

Usage:

```diff
use Bmatovu\MtnMomo\Products\Collection;

$collection = new Collection();

+ $collection->setCurrency('UGX');

// Request a user to pay
$collection->transact('EXT_REF_ID', '07XXXXXXXX', 100);
```

**Comment by GENL** [comment]
Good job!
